v2/internal/controllers/generic_controller.go

```diff
@@ -134,6 +143,7 @@ func register(
 	reconciledResourceLookup map[schema.GroupKind]schema.GroupVersionKind,
 	clientFactory ARMClientFactory,
 	info registration.StorageType,
+	extensionsMap map[schema.GroupVersionKind]genruntime.ResourceExtension,
```

Comment:
Similarly, perhaps call this just `extensions`.
```suggestion
	extensions map[schema.GroupVersionKind]genruntime.ResourceExtension,
```